### ISSUE TITLE
Problem matcher example 

### ISSUE THREAD

**Issue by stefanbuck** [rkusa/jest-action#4]
Are you aware of any existing Problem Matcher for Jest? Jest’s output  isn't really suitable for parsing or at least not that simple. I proposed adding [xpath support](https://github.com/actions/toolkit/issues/260) in order to make this simpler

**Comment by rkusa** [comment]
I am not aware of any, but I haven't really look for one yet. Just a thought from the top of my head, maybe one could output the Jest result as JSON (`--testLocationInResults --json`) and reformat it via the `jq` command line tool to make it easier to parse for a problem matcher 🤔

**Comment by stefanbuck** [comment]
Yeah, I was thinking about something similar.  I will see if I find time to look into this

**Comment by rkusa** [comment]
Or, reformat the JSON into the following output, which will also be displayed as annotations: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/development-tools-for-github-actions#set-an-error-message-error

```
::error file={name},line={line},col={col}::{message}
```

**Comment by stefanbuck** [comment]
Thanks for sharing. I didn't had time to look into this, but this looks really interesting.
